src/graphics/gfx_update.cpp

```diff
@@ -2609,9 +2610,9 @@ void UpdateGraphics(bool skipRepaint)
 //                    BitBlt myBackBuffer, .X - vScreen(Z).Left, .Y - vScreen(Z).Top, 32, 32, GFX.ECursor(2).hdc, 0, 0, vbSrcPaint
                     XRender::renderTexture(curX, curY, GFX.ECursor[2]);
 //                    If LCase(frmLayers.lstLayer.List(frmLayers.lstLayer.ListIndex)) <> "default" Then
-                    if(!e.Layer.empty() && SDL_strcasecmp(e.Layer.c_str(), "Default") != 0)
+                    if(e.Layer != LAYER_NONE && e.Layer != LAYER_DEFAULT)
 //                        SuperPrint UCase(frmLayers.lstLayer.List(frmLayers.lstLayer.ListIndex)), 3, .X + 28, .Y + 34
-                        SuperPrint(e.Layer, 3, X + 28, Y + 34);
+                        SuperPrint(Layer[e.Layer].Name, 3, X + 28, Y + 34);
```

Comment:
The same as on the world side, is that valid to use the X and Y instead of `curX` and `curY`? At this line, looks like my own bug because I did this mess. Could you verify?

Reply:
I haven't backported the new editor drawing code yet, but I'll change these to `curX` and `curY` for now. The new editor drawing code also used `curX` and `curY` here.